fmriprep/cli/run.py

```diff
@@ -786,7 +786,18 @@ def build_workflow(opts, retval):
     boilerplate = retval['workflow'].visit_desc()
 
     if boilerplate:
-        (logs_path / 'CITATION.md').write_text(boilerplate)
+        # To please git-annex users and also to guarantee consistency
+        # among different renderings of the same file, first remove any
+        # existing one
+        citation_files = {
+            ext: str(logs_path / 'CITATION.' + ext)
+            for ext in ('bib', 'tex', 'md', 'html')
+        }
+        for citation_file in citation_files.values():
+            if op.lexists(citation_file):
```

Comment:
```suggestion
            if citation_file.exists() or citation_file.is_symlink():
```

Reply:
why bother with two instead of a single one?

Reply:
I'm trying to get rid of all `os.path` calls. Additionally, I think this is more explicit - I'm sure I'll read `op.exists` instead of `op.lexists` in just two months from now.